src/scss/base/_mixin.scss

```diff
@@ -0,0 +1,15 @@
+@mixin telefono {
```

Comment:
Try to write all your functions, variables, mixins, etc. names in just one language is not a good practice to mix Spanish with English or vice versa